scapy/utils.py

```diff
@@ -1636,12 +1639,21 @@ def _read_block_shb(self):
             raise EOFError
 
         blocklen = struct.unpack(self.endian + "I", _blocklen)[0]
-        if blocklen < 16:
-            warning("Invalid SHB block length!")
+        if blocklen < 28:
```

Comment:
Can you explain this change? Is it an error in the current implementation?

Reply:
From [SHB specification ](https://ietf-opsawg-wg.github.io/draft-ietf-opsawg-pcap/draft-ietf-opsawg-pcapng.html#name-section-header-block ) i understand the mandatory fields add up to 28 bytes.

Reply:
You are right this is a mistake.